.gitignore

```
@@ -0,0 +1,3 @@
+node_modules/
+test.md/
```

Your `test.md` file is still appearing on Github. To ignore the `test.md` file you just need to past the name of the file like that: `test.md`. In this case, you are having a typo issue `test.md/`.   Kindly remove the forward-slash to avoid this issue. 😊
```suggestion
test.md
```